packages/js/components/src/experimental-tree-control/stories/index.tsx

```diff
@@ -33,7 +33,14 @@ const listItems: Item[] = [
 export const SimpleTree: React.FC = () => {
 	return (
 		<BaseControl label="Simple tree" id="simple-tree">
-			<TreeControl id="simple-tree" items={ listItems } />
+			<TreeControl
+				id="simple-tree"
+				items={ listItems }
+				onFirstTreeItemBack={ () =>
+					// eslint-disable-next-line no-alert
+					alert( 'onFirstTreeItemBack called' )
```

Comment:
Also, none of the other stories have been updated to make use of this, including `SelectTreeControl` ones. Since I don't know why this is needed, I can't determine if the other stories should be updated to do something with this or not.